addon/components/bs-form-element.js

```diff
@@ -172,6 +172,15 @@ export default FormGroup.extend({
   placeholder: null,
 
   /**
+   * Control element's HTML5 autofocus attribute
+   *
+   * @property autofocus
+   * @type string
+   * @public
+   */
+  autofocus: null,
```

Comment:
This should be a boolean property according to HTML5 specs. Needs updating the example in the docs as well.